ui-tests/cypress/integration/content/management.spec.js

```diff
@@ -137,4 +163,36 @@ describe('Contents', () => {
     });
     cy.contentTypesController().then(controller => controller.deleteContentType(contentTypeCode));
   });
+
+
+  describe('Add content', () => {
+    it('Add content', () => {
+      cy.intercept('POST', contentsAPIUrl, (req) => {
```

Comment:
Why it is needed to intercept the request instead of just getting the payload when the request is performed?

Reply:
The 'Add content' test doesn't directly consume the contents API. It creates a content via the UI. So as far as I know, that's the easiest way to get the corresponding content 'id' or 'code' after it has been added.

Reply:
if you create the content via UI, you should be able to use what you have defined as part of the creation to identify it (id, name, values, etc...); is the content id something different? if so, why do you need that specifically?

Reply:
The content id is generated by the server once it's created. It's used for the test cleanup (i.e. deleting the content).